RIP-0016: doing RESADD with existing resource id will replace an existing resource (#53)
rmrk-team/rmrk-spec: standards/rmrk2.0.0/interactions/resadd.md

```txt
rmrk::RESADD::2.0.0::5105000-0aff6865bed3a66b-DLEP-DL15-00000001::%7B%22id%22%3A%22V1i6B%22%2C%src%22%3A%22hash-of-metadata-guest-bird-art-with-jetpack%22%2C%22metadata%22%3A%22hash-of-metadata-with-credits%22%7D
```
## Replacing an existing resource

> If a resource `id` is matching the existing resource `id` on that NFT, then the resource gets replaced with a new resource fields

Reviewer: ```suggestion
If the resource `id` matches an existing resource `id` on that NFT, then the existing resource gets replaced with the new one.

```
